docs/en/latest/plugins/datadog.md

```diff
@@ -23,62 +29,93 @@ title: datadog
+```
 
-## Exported Metrics
+## Exported metrics
 
-Apache APISIX agent, for every request response cycle, export the following metrics to DogStatsD server if the datadog plugin is enabled:
+When the `datadog` Plugin is enabled, APISIX agent exports the following metrics to the DogStatsD server for each request/response cycle:
 
-| Metric Name               | StatsD Type   | Description               |
-| -----------               | -----------   | -------                   |
-| Request Counter           | Counter       | Number of requests received.   |
-| Request Latency           | Histogram     | Time taken to process the request (in milliseconds). |
-| Upstream latency          | Histogram     | Time taken to proxy the request to the upstream server till a response is received (in milliseconds). |
-| APISIX Latency            | Histogram     | Time taken by APISIX agent to process the request (in milliseconds). |
-| Ingress Size              | Timer         | Request body size in bytes. |
-| Egress Size               | Timer         | Response body size in bytes. |
+| Metric name      | StatsD type | Description                                                                                           |
+|------------------|-------------|-------------------------------------------------------------------------------------------------------|
+| Request Counter  | Counter     | Number of requests received.                                                                          |
+| Request Latency  | Histogram   | Time taken to process the request (in milliseconds).                                                  |
+| Upstream latency | Histogram   | Time taken to proxy the request to the upstream server till a response is received (in milliseconds). |
+| APISIX Latency   | Histogram   | Time taken by APISIX agent to process the request (in milliseconds).                                  |
+| Ingress Size     | Timer       | Request body size in bytes.                                                                           |
+| Egress Size      | Timer       | Response body size in bytes.                                                                          |
 
 The metrics will be sent to the DogStatsD agent with the following tags:
 
-> If there is no suitable value for any particular tag, the tag will simply be omitted.
+- `route_name`: Name specified in the Route schema definition. If not present or if the attribute `prefer_name` is set to false, falls back to the Route ID.
+- `service_name`: If a Route has been created with an abstracted Service, the Service name/ID based on the attribute `prefer_name`.
+- `consumer`: If the Route is linked to a Consumer, the username will be added as a tag.
+- `balancer_ip`: IP address of the Upstream balancer that processed the current request.
+- `response_status`: HTTP response status code.
+- `scheme`: Request scheme such as HTTP, gRPC, and gRPCs.
```

Comment:
I guess it would be nice to put these into a tabular form. What do you think?

Reply:
I think we are following this structure without tables in some other places as well. So, we can keep it as it is now.